### DISCUSSION TITLE
Page summarization doesn't work in page mode?

### DISCUSSION THREAD

**Discussion by gerroon** [n4ze3m/page-assist#280] [Q&A] [answered]
Hi, 
Using it with Firefox Windows. I open a web page, enable "chat with current page" and fire the question below. The response is a bit weird. It would be nice if summarizing is  also possible in the page chat mode. thanks

`summarize the page`

```
llama3.2:latest

I don't have any information about tools used to summarize webpages in this context. The provided text discusses a social media post and

online reactions related to immigration, billionaires, but it doesn't mention summarizing webpages or specific tools for doing so. If you'd 

like to provide more context or clarify your question, I'll do my best to help.
```

**Comment by n4ze3m** [comment]
Hey, sorry about that. Are you using chat with the current page with RAG, or without RAG?

![image](https://github.com/user-attachments/assets/f571f3d5-8412-42cd-a0c1-d443152224b3)

If you are using it with RAG, turn it off and try the same message. It produces a better summarization

**Reply by gerroon** [reply]
Thanks I will test it. What is the difference between the Normal and RAG options as far as embedding a page goes?

**Reply by n4ze3m** [reply, marked as answer]
In the default RAG case, it will perform a vector search and then send the results to the LLM, which may not have the full information for summarization. Without RAG, however, it sends all the text content from the current site to the LLM, making it better suited for summarization tasks.
